src/visualization/types/SingleStat/view.tsx

```diff
@@ -2,15 +2,22 @@
 import React, {FC} from 'react'
 
 // Utils
-import LatestValueTransform from 'src/visualization/components/LatestValueTransform'
 import {generateThresholdsListHexs} from 'src/shared/constants/colorOperations'
-import {formatStatValue} from 'src/visualization/utils/formatStatValue'
 
 // Types
 import {SingleStatViewProperties} from 'src/types/dashboards'
 import {VisualizationProps} from 'src/visualization'
 
+import {
+  Config,
+  Plot,
+  getLatestValues,
+  formatStatValue,
+} from '@influxdata/giraffe'
+import LatestValueTransform from '../../components/LatestValueTransform'
```

Comment:
One more nitpick and this PR will be good to go. This import should not be changed to use relative paths. Can you please restore the original line that was deleted above under "// Utils" ?

Reply:
Yes, just did, sorry about that, somehow slipped.